tests/bsim/bluetooth/mesh/tests_scripts/blob_mdls/blob_srv_persistence.sh

```diff
@@ -36,27 +36,27 @@ RunTestFlash blob_recover_phase \
    blob_cli_stop -flash_rm blob_srv_stop -flash_rm -- -argstest expected-phase=4
 
 # The same test but with PSA crypto
-overlay="overlay_pst_conf_overlay_psa_conf"
+overlay="overlay_pst_conf_overlay_ss_conf_overlay_psa_conf"
```

Comment:
Change the comment as well?

Reply:
comment is correct and doesn't require changes